skrf/networkSet.py

```diff
@@ -794,14 +797,15 @@ def uncertainty_ntwk_triplet(self, attribute: str, n_deviations: int = 3):
         Example
         -------
         >>> (ntwk_mean, ntwk_lb, ntwk_ub) = my_ntwk_set.uncertainty_ntwk_triplet('s')
+        >>> (ntwk_mean, ntwk_lb, ntwk_ub) = my_ntwk_set.uncertainty_ntwk_triplet('s_mag')
 
         """
         ntwk_mean = self.__getattribute__('mean_'+attribute)
         ntwk_std = self.__getattribute__('std_'+attribute)
         ntwk_std.s = n_deviations * ntwk_std.s
 
-        upper_bound = (ntwk_mean + ntwk_std)
-        lower_bound = (ntwk_mean - ntwk_std)
+        upper_bound = (ntwk_mean + ntwk_std.s)
+        lower_bound = (ntwk_mean - ntwk_std.s)
```

Comment:
I've tested it and found it was the same as the previous implementation. I'm interested if you have some counter-example.
```suggestion
        upper_bound = (ntwk_mean + ntwk_std)
        lower_bound = (ntwk_mean - ntwk_std)
```

Reply:
Committed